grpc/shared_conn.go

```diff
@@ -156,7 +156,7 @@ func (sc *SharedConn) ResetConn(conn rpc.ClientConn, moduleLogger logging.Logger
 		// The first call to `ResetConn` happens before anything can access `sc.logger`. So long as
 		// we never write to the member variable, everything can continue to access this without
 		// locks.
-		sc.logger = moduleLogger.Sublogger("conn")
+		sc.logger = moduleLogger.Sublogger("network_outgoing.conn")
```

Comment:
From what I understand, this is where `sc.logger` gets initialized. I think it is valuable to keep using the `moduleLogger` for the module logger name prefix, but since this logger deals with outgoing network connections, we want to be able to configure this using the `network_outgoing` string as well. This deviates from the `logging.GetOrNewLogger("rdk.network_outgoing")` pattern slightly in order to keep the module name and still be configurable using e.g. `*.network_outgoing.*`. I am open to suggestions or reverting this change.

Reply:
Thanks for calling this out. So, logs from the shared conn to a module from the module manager will have the logger name: `rdk.modmanager.[modulename].network_outgoing.conn`? I think that's fine. I can't really think of a better option: `rdk.network_outgoing.modmanager.module.conn` would _also_ be slightly inconvenient, as if a user tried to set `rdk.modmanager.*`, they'd miss this logger. Also tagging @dgottlieb for visibility on our changes here.

Reply:
correct, I also agree this is probably the best we can do for now